system-addon/content-src/styles/_theme.scss

```diff
@@ -53,7 +53,7 @@
   --newtab-background-color: $grey-80;
   --newtab-border-primary-color: $grey-10-80;
   --newtab-border-secondary-color: $grey-10-20;
-  --newtab-button-primary-color: $blue-40;
+  --newtab-button-primary-color: $blue-60;
```

Comment:
I sneaked this button color change in here. Basically we realized we need blue-60 when it is background color for light text in order to hit the contrast minimum.